pkg/store/etcd.go

```diff
@@ -99,13 +86,21 @@ func (s *Etcd) Get(key string) (*KVPair, error) {
 }
 
 // Put a value at "key"
-func (s *Etcd) Put(key string, value []byte) error {
-	if _, err := s.client.Set(key, string(value), 0); err != nil {
+func (s *Etcd) Put(key string, value []byte, opts *RequestOptions) error {
+
+	// Default TTL = 0 means no expiration
+	ttl := 0 * time.Second
+	if opts != nil && opts.Ephemeral {
+		ttl = 60 * time.Second
```

Comment:
If I put this into `WriteOption` that would defeat the purpose of hiding it behind the `Ephemeral` flag. Same thing if I put this in Config. Best bet for now would be to put those as `const` (unless we can specify `heartbeat` + `refresh` time in the `cli`).